docs/training/working-with-data-and-model-artifacts.md

```diff
@@ -0,0 +1,87 @@
+(working-with-data-and-model-artifacts)=
+# Working with Data and Model Artifacts
+
+When running a training job, you will need to pass in data to train and save the resulting model. Both the data and model can be considered [Artifacts](https://docs.mlrun.org/en/latest/store/artifacts.html) in MLRun. In the context of an ML pipeline, the data would be an `input` and the model would be an `output`.
```

Comment:
When running a training job, you need to pass in data to train and save the resulting model. Both the data and model can be considered [Artifacts](../store/artifacts.html) in MLRun. In the context of an ML pipeline, the data is an `input` and the model is an `output`.